fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -289,7 +289,7 @@ protected Action buildActionChain() {
             // system-generated value. For the update-or-create scenario, see update().
             // Default version is 1 for a brand new FHIR Resource.
             int newVersionNumber = 1;
-            logicalId = logicalIdentityProvider.createNewIdentityValue();
+            logicalId = getLogicalId();
```

Comment:
What does this change get us?

Reply:
getLogicalId was added to the FHIRPersistence interface, so that it can be called from outside the PL.  basically, he made that piece of it public.
so now the FHIRRestHelper can delegate to the FHIRPersistence to have it generate a resource id (separate from actually creating a resource)

Reply:
a-ok